types/monome-grid/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for monome-grid 1.0
+// Project: https://github.com/dinchak/node-monome-grid
+// Definitions by: Arlan Jaska <https://github.com/ajaska>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+export interface MonomeGrid {
+  refresh(leds: number[][]): void;
+  key(keyHandler: (x: number, y: number, s: number) => void): void;
+}
+
+export default function monomeGrid(serialNumber?: string): Promise<MonomeGrid>;
```

Comment:
It looks like the underlying library exports a module object.  To match that behavior, I think you'd need to use the  `exports =` syntax.